values/velero/velero.gotmpl

```diff
@@ -67,8 +69,10 @@ podSecurityContext:
   runAsUser: 1000
 
 configuration:
-  {{- if eq $sp.type "azureBlob" }}
+  uploaderType: restic
```

Comment:
Does it mean rustic will be always used?
If so do we need update in otomi.io?

Reply:
Not exactly. In the most velero versions there is an alternative to restic(kopia). With this option we choose one over the other. Enabling the [Node Agent](https://velero.io/docs/main/file-system-backup/#install-velero-node-agent)(aka. File System Backup) is done via the `deployNodeAgent` parameter. What we can do maybe in a later stage is to accommodate this choice in otomi(restic or kopia).

Reply:
ah now, I get it. Could you please add this info as a comment in code?

Reply:
Done